app-misc/c_rehash/c_rehash-1.7_p20160204.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_FILE="openssl-c_rehash.sh"
+GIT_COMMIT="f79908c1f7004d0abd384451abd923c6b6ba323d"
+
+DESCRIPTION="c_rehash script from OpenSSL"
+HOMEPAGE="https://www.openssl.org/ http://git.pld-linux.org/?p=packages/openssl.git https://github.com/pld-linux/openssl/"
+SRC_URI="http://git.pld-linux.org/?p=packages/openssl.git;a=blob_plain;f=${MY_FILE};hb=${GIT_COMMIT} -> ${MY_FILE}.${PV}"
```

Comment:
PID-Linux.org is available via HTTPS. Please Update HOMEPAGE and SRC_URI.

Reply:
Sorry for that. Updated.